doc/source/user_guide/advanced.rst

```diff
@@ -179,18 +179,18 @@ on a deeper level.
 
 .. _advanced.shown_levels:
 
-Defined Levels
+Defined levels
 ~~~~~~~~~~~~~~
 
-The :class:`MultiIndex` keeps all the defined levels of an index, even
+The repr of a ``MultiIndex`` shows all the defined levels of an index, even
```

Comment:
Not sure if this is from a merge, but we avoid having unrelated changes in PRs, in case something goes wrong and we need to revert. If this comes from a merge that applies it, please revert to the original. If it's you proposing to change those, that's welcome, but please open a separate PR for anything that is not about fixing the capitalization of titles.